build.gradle

```diff
@@ -3,7 +3,7 @@ ext.buildTime = new Date().toInstant()
 buildscript {
     ext.springBootVersion = '3.+'
     ext.springCloudVersion = '2023.+'
-    ext.springDataVersion = '2023.+'
+    ext.springDataVersion = '2024.+'
```

Comment:
Since this change can be trickier than it seems, I would like to do this separately.